package.json

```diff
@@ -14,11 +14,20 @@
     "owners-file": "https://github.com/bkeepers/owners-node.git"
   },
   "devDependencies": {
+    "expect": "^1.20.2",
+    "mocha": "^3.2.0",
     "xo": "^0.16.0"
   },
   "xo": {
     "esnext": true,
-    "space": true
+    "space": true,
+    "envs": [
+      "node",
+      "mocha"
+    ],
+    "rules": {
+      "camelcase": 1
+    }
```

Comment:
I'm starting to wonder if these rules should be pulled out into a common config for all probot plugins somewhere.